rclcpp/include/rclcpp/experimental/subscription_intra_process_buffer.hpp

```diff
@@ -91,6 +93,7 @@ class SubscriptionIntraProcessBuffer : public SubscriptionROSMsgIntraProcessBuff
       buffer_type,
       qos_profile,
       std::make_shared<Alloc>(subscribed_type_allocator_));
+    TRACEPOINT(IPB_to_subscription, buffer_.get(), static_cast<const void *>(this));
```

Comment:
As per my suggestions in the `ros2_tracing` PR. Also, there's a missing `static_cast`.

```suggestion
    TRACEPOINT(
      ipb_to_subscription,
      static_cast<const void *>(buffer_.get()),
      static_cast<const void *>(this));
```

Reply:
fixed in [9ead575](https://github.com/ros2/rclcpp/pull/2091/commits/9ead5753751ed694c7acac47483c6f37ef61919d).